### PR TITLE
Change Cron Binding from `input/output` to `input` binding

### PR DESCRIPTION
**Pull request by sarsharma**
Signed-off-by: Sarthak Sharma <sartsharma@microsoft.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Changing cron binding to input only binding to certify it as a stable component 
See linked issue and Certification Tests PR for reference

Also fix Kubernetes Events Binding wrongly listing having output binding support

## Issue reference

https://github.com/dapr/components-contrib/issues/2308
https://github.com/dapr/components-contrib/pull/2191/

### CODE REVIEW

File: daprdocs/data/components/bindings/generic.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   since: "1.0"
```
Comment (mukundansundar) [inline daprdocs/data/components/bindings/generic.yaml]: Change since and state also to Beta or stable ?? From 1.10

**Review by mukundansundar — APPROVED** [review]

**Review by mukundansundar — CHANGES_REQUESTED** [review]
Change base to 1.10

**Comment by sarsharma** [comment]
Changed base to v1.10 , DCO seems to be failing due to older commits in v1.10 branch

**Comment by mukundansundar** [comment]
~~Can you rebase with a single commit on top of v1.10....?~~

**Review by mukundansundar — APPROVED** [review]

**Review by hhunter-ms — APPROVED** [review]
lgtm

**Pull request merged**